exporters/common/src/main/java/io/opentelemetry/exporter/internal/FailedExportException.java

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.exporter.internal;
+
+import io.opentelemetry.exporter.internal.grpc.GrpcResponse;
+import io.opentelemetry.exporter.internal.http.HttpSender;
+import javax.annotation.Nullable;
+
+/**
+ * Represents the failure of a gRPC or HTTP exporter.
+ *
+ * <p>This class is internal and is hence not for public use. Its APIs are unstable and can change
+ * at any time.
+ */
+public abstract class FailedExportException extends Exception {
+
+  private static final long serialVersionUID = 6988924855140178789L;
+
+  private FailedExportException(@Nullable Throwable cause) {
+    super(cause);
+  }
+
+  /** Indicates an HTTP export failed after receiving a response from the server. */
+  public static HttpExportException httpFailedWithResponse(HttpSender.Response response) {
+    return new HttpExportException(response, null);
+  }
+
+  /** Indicates an HTTP export failed exceptionally without receiving a response from the server. */
+  public static HttpExportException httpFailedExceptionally(Throwable cause) {
+    return new HttpExportException(null, cause);
+  }
+
+  /** Indicates an HTTP export failed after receiving a response from the server. */
```

Comment:
```suggestion
  /** Indicates an gRPC export failed after receiving a response from the server. */
```
copy paste error on my part

Reply:
Oops - easy fix